openghg/store/_footprints.py

```diff
@@ -313,8 +314,13 @@ def read_file(
         if not filepath:
             return {}
 
-        if chunks is None:
-            chunks = {}
+        chunks = self.check_chunks(
```

Comment:
we need to use the species to update `high_time_resolution` etc before this step